great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -1491,18 +1501,19 @@ def add_checkpoint(
             slack_webhook=slack_webhook,
             notify_on=notify_on,
             notify_with=notify_with,
-            ge_cloud_id=ge_cloud_id,
-            expectation_suite_ge_cloud_id=expectation_suite_ge_cloud_id,
+            ge_cloud_id=id,
+            expectation_suite_ge_cloud_id=expectation_suite_id,
             default_validation_id=default_validation_id,
         )
 
-        self.checkpoint_store.add_checkpoint(checkpoint, name, ge_cloud_id)
+        self.checkpoint_store.add_checkpoint(checkpoint=checkpoint, name=name, id=id)
         return checkpoint
 
     def get_checkpoint(
         self,
-        name: Optional[str] = None,
-        ge_cloud_id: Optional[str] = None,
+        name: str | None = None,
+        ge_cloud_id: str | None = None,
+        id: str | None = None,
```

Comment:
Doesn't need to be in this PR but can we add overloads for these methods that take either `name` or `id`?

Reply:
adding a ticket that adds overloads to all the new crud methods!